Flashcards.harris-andy/Classes/UseDB.cs

```diff
@@ -0,0 +1,133 @@
+using Microsoft.Data.SqlClient;
+using Dapper;
+using Flashcards.harris_andy.Classes;
+
+namespace Flashcards.harris_andy;
+
+public class UseDB
+{
+    public void InitializeDatabase()
+    {
+        string connectionStringMaster = "Server=localhost,1433;Database=master;User Id=SA;Password=SuperStrongSexyPassword123;TrustServerCertificate=True;";
+        using var connectionMaster = new SqlConnection(connectionStringMaster);
+        var checkDbQuery = @"
+                IF NOT EXISTS (SELECT name FROM sys.databases WHERE name = 'FlashCards')
+                    BEGIN
+                    CREATE DATABASE FlashCards;
```

Comment:
I didn't consider the option of the user naming the db. I didn't think it mattered because all references to the db name were baked in. To your point in another comment though, I'll try to tear this out and give the user a short script to create their own database.